app/services/sync_service/library_hours.rb

```diff
@@ -25,8 +25,7 @@ def self.call
                   { coordinates: "Sheet2!L2:L", slug: "ambler" },
                   { coordinates: "Sheet2!M2:M", slug: "ginsburg" },
                   { coordinates: "Sheet2!N2:N", slug: "podiatry" },
-                  { coordinates: "Sheet2!O2:O", slug: "innovation" },
```

Comment:
@cdoyle-temple You intended to remove the innovation space here but nowhere else?

Reply:
@bibliotechy The innovation space hasn't been removed at all.

Reply:
Never mind, I'm daft